### ISSUE TITLE
Unable to mount one of your folders. Please check your folders in Homestead.yaml

### ISSUE THREAD

**Issue by Direclaw** [laravel/homestead#830]
- Vagrant:  2.0.3
  - Provider: Virtualbox 5.2.8 r121009 (Qt5.6.2)
  - Homestead: 7.3.0 (master)
  - Windows 10
  - homestead.yalm is unaltered
---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: ~/code
      to: /home/vagrant/code

sites:
    - map: homestead.test
      to: /home/vagrant/code/public

databases:
    - homestead

https://gist.github.com/Direclaw/034cb7e033b2f59e9a22dd84e277c879

Code folder should show up

Instead the folder does not show up but everything else works normally

To reproduce
1. Install vagrant
2. Add laravel/homestead box
3. Clone the homestead repository
4. Run init.sh
5. Vagrant up

**Comment by svpernova09** [comment]
In your mapped folder change ~/code to C:\Users\Gabriel\code

--
- Joe Ferguson
JoeFerguson.me
MemphisPHP.org

**Comment by Direclaw** [comment]
I tried that before and the result is the same. I'm a few days struggling with this.

**Comment by svpernova09** [comment]
The path is the issue if you look at the up log it only mounts /vagrant. Should see that and all
Folder map

--
- Joe Ferguson
JoeFerguson.me
MemphisPHP.org

**Comment by Direclaw** [comment]
Tried again using this homestead.yaml

---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: C:\Users\Gabriel\code
      to: /home/vagrant/code

sites:
    - map: homestead.test
      to: /home/vagrant/code/public

databases:
    - homestead


here is the result

https://gist.github.com/Direclaw/b1978d487e30232c4b346296f6530325

**Comment by svpernova09** [comment]
That means C:\Users\Gabriel\code isn’t a folder or valid path

--
- Joe Ferguson
JoeFerguson.me
MemphisPHP.org

**Comment by Direclaw** [comment]
Ok, I think I got it:

The folder you are trying to link needs to exist in advance otherwise that message appears. I'm not sure if this is a new thing because I don't remember it but I could have forgotten it.

I created the folder, destroyed, upped and it is working.

Is this normal behavior? I didn't found anything about this anywhere. well, anyway, it is working now.

I really appreciated the help.
